Description: Add testing.sh
 Add testing.sh which will be installed to /usr/bin/.
 .
 hello (2.10-3leaxyu) noble; urgency=medium
 .
   * Add testing.sh which will be installed to /usr/bin/.
Author: Xiaoyu LI <leaxyu@msn.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2025-05-25

--- /dev/null
+++ hello-2.10/testing.sh
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "this is a test from Xiaoyu LI" >&2
+
